rest_framework_mongoengine/fields.py

```diff
@@ -85,6 +86,10 @@ class ReferenceField(DocumentField):
     We always dereference DBRef object before serialization
     TODO: Maybe support DBRef too?
     """
```

Comment:
I do not know if 2 different errors are needed. I kept them different because it helped me debug an issue in my own code.